src/timeline/app.js

```diff
@@ -51,32 +64,29 @@ $(document).ready(function () {
 
     track_controls.height(new_track_height);
     $("#scrolling_tracks").height(new_track_height);
-    body_object.scope().playhead_height = $("#track-container").height();
-    $(".playhead-line").height(body_object.scope().playhead_height);
+      body_object.scope().playhead_height = $("#track-container").height();
+      $(".playhead-line").height(body_object.scope().playhead_height);
   });
 
-  // Check for Qt Integration
-  if (typeof timeline !== "undefined") {
-    timeline.qt_log("Qt Found!");
-    body_object.scope().enableQt();
-    timeline.page_ready();
-    body_object.scope().setThumbAddress(timeline.get_thumb_address());
-  }
-
-  // Manually trigger the window resize code (to verify it runs at least once)
-  $(window).trigger("resize");
-
   // Bind to keydown event (to detect SHIFT)
   body_object.keydown(function (event) {
     if (event.which === 16) {
+      if (timeline) {
+        timeline.qt_log2("DEBUG", "Shift pressed!");
+      }
       body_object.scope().shift_pressed = true;
     }
   });
 
   body_object.keyup(function () {
```

Comment:
So, here's where I'm an idiot. Impressive that I expect the keyup function to inspect `event` without _passing it in_. DUH.
```suggestion
  body_object.keyup(function (event) {
```